automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -136,6 +136,7 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
           val secondApiCall =
             Leonardo.cluster.getRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
           firstApiCall.auditInfo.dateAccessed should be < secondApiCall.auditInfo.dateAccessed
+          Leonardo.cluster.stopRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
```

Comment:
this isn't necessary, but doesn't hurt to have it here just so we have welder log flushed to staging bucket